roadmap/implementers-guide/src/node/approval/approval-distribution.md

```diff
@@ -0,0 +1,197 @@
+# Approval Distribution
+
+A subsystem for the distribution of assignments and approvals for approval checks on candidates over the network.
+
+The [Approval Voting](approval-voting.md) subsystem is responsible for active participation in a protocol designed to select a sufficient number of validators to check each and every candidate which appears in the relay chain. Statements of participation in this checking process are divided into two kinds:
+  - **Assignments** indicate that validators have been selected to do checking
+  - **Approvals** indicate that validators have checked and found the candidate satisfactory.
+
+The [Approval Voting](approval-voting.md) subsystem handles all the issuing and tallying of this protocol, but this subsystem is responsible for the disbursal of statements among the validator-set.
+
+The inclusion pipeline of candidates concludes after availability, and only after inclusion do candidates actually get pushed into the approval checking pipeline. As such, this protocol deals with the candidates _made available by_ particular blocks, as opposed to the candidates which actually appear within those blocks, which are the candidates _backed by_ those blocks. Unless stated otherwise, whenever we reference a candidate partially by block hash, we are referring to the set of candidates _made available by_ those blocks.
+
+We implement this protocol as a gossip protocol, and like other parachain-related gossip protocols our primary concerns are about ensuring fast message propagation while maintaining an upper bound on the number of messages any given node must store at any time.
+
+Approval messages should always follow assignments, so we need to be able to discern two pieces of information based on our [View](../../types/network.md#universal-types):
+  1. Is a particular assignment relevant under a given `View`?
+  2. Is a particular approval relevant to any assignment in a set?
+
+These two queries need not be perfect, but they must never yield false positives. For our own local view, they must not yield false negatives. When applied to our peers' views, it is acceptable for them to yield false negatives. The reason for that is that our peers' views may be beyond ours, and we are not capable of fully evaluating them. Once we have caught up, we can check again for false negatives to continue distributing.
+
+For assignments, what we need to be checking is whether we are aware of the (block, candidate) pair that the assignment references. For approvals, we need to be aware of an assignment by the same validator which references the candidate being approved.
```

Comment:
"..references, or the candidate equivocation in the case of an equivocation in relay chain block production".

Yes, I know we're not doing those yet, but I wanted to make a note.

Reply:
All of the text you wrote about approvals is in a document called `protocol-approvals` in the guide, and it mentions this. It's less confusing to mention only stuff that's implemented here, IMO.